### PR TITLE
add prometheus metrics for the local api

### PR DESCRIPTION
**Pull request by buixor**
- Metrics for `machines` and `bouncers` : count requests per routes/method
 - `aggregated` Prometheus mode will not expose the hits-per-method-per-route-per-machine/bouncer metrics to keep cardinality low
 - `cscli metrics` displays the metrics like this :

```bash
INFO[0000] Local Api Metrics:                           
+--------------------+--------+------+
|       ROUTE        | METHOD | HITS |
+--------------------+--------+------+
| /v1/alerts         | GET    |    3 |
| /v1/decisions      | DELETE |    1 |
| /v1/decisions      | GET    |    4 |
| /v1/watchers/login | POST   |    6 |
+--------------------+--------+------+
INFO[0000] Local Api Machines Metrics:                  
+---------+---------------+--------+------+
| MACHINE |     ROUTE     | METHOD | HITS |
+---------+---------------+--------+------+
| test    | /v1/decisions | DELETE |    1 |
| test    | /v1/alerts    | GET    |    3 |
+---------+---------------+--------+------+
INFO[0000] Local Api Bouncers Metrics:                  
+---------+---------------+--------+------+
| BOUNCER |     ROUTE     | METHOD | HITS |
+---------+---------------+--------+------+
| lol     | /v1/decisions | GET    |    4 |
+---------+---------------+--------+------+
INFO[0000] Local Api Bouncers Decisions:                
+---------+---------------+-------------------+
| BOUNCER | EMPTY ANSWERS | NON-EMPTY ANSWERS |
+---------+---------------+-------------------+
| lol     |             1 |                 3 |
+---------+---------------+-------------------+

```

### CODE REVIEW

**Comment by AlteredCoder** [comment]
Can we display the table only if there is metrics please?

File: cmd/crowdsec-cli/metrics.go [hunk 1]
```diff
@@ -235,6 +300,10 @@ func ShowPrometheus(url string) {
 		parsersTable.Render()
 		log.Printf("Local Api Metrics:")
 		apilTable.Render()
+		log.Printf("Local Api Machines Metrics:")
+		apilMachinesTable.Render()
+		log.Printf("Local Api Bouncers Metrics:")
+		apilBouncersTable.Render()
 	} else if csConfig.Cscli.Output == "json" {
 		for _, val := range []map[string]map[string]int{acquis_stats, parsers_stats, buckets_stats, apil_stats} {
```
Comment (AlteredCoder) [inline cmd/crowdsec-cli/metrics.go:308]: I think we want to add the news stats here ?

Comment (buixor) [reply cmd/crowdsec-cli/metrics.go]: yes

**Review by erenJag — COMMENTED** [review]
Change `apil` to `lapi` please :)

**Review by AlteredCoder — APPROVED** [review]

**Pull request merged**